bundles/org.openhab.binding.visualcrossing/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,1995 @@
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="solar-energy-channel">
+		<item-type>Number</item-type>
+		<label>Solar Energy</label>
+		<description>(MJ /m^2) Indicates the total energy from the sun that builds up over an hour or day. See the full
+			solar
+			radiation data documentation
+			(https://www.visualcrossing.com/resources/documentation/weather-data/how-to-obtain-solar-radiation-data/)
+			and Wind and
+			Solar Energy
+			(https://www.visualcrossing.com/resources/documentation/weather-api/energy-elements-in-the-timeline-weather-api/)
+			pages.
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="conditions-channel">
+		<item-type>String</item-type>
+		<label>Conditions</label>
+		<description>Textual representation of the weather conditions. See Weather Data Conditions
+			(https://www.visualcrossing.com/resources/documentation/weather-api/weather-condition-fields/).
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="icon-channel">
+		<item-type>String</item-type>
+		<label>Icon</label>
+		<description>A fixed, machine readable summary that can be used to display an icon</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="stations-channel" advanced="true">
+		<item-type>String</item-type>
+		<label>Stations</label>
+		<description>The weather stations (comma separated) used when collecting an historical observation record
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="source-channel" advanced="true">
+		<item-type>String</item-type>
+		<label>Source</label>
+		<description>
+			The type of weather data used for this weather object. – Values include historical observation (“obs”),
+			forecast (“fcst”), historical forecast (“histfcst”) or statistical forecast (“stats”). If multiple types are used in
+			the same day, “comb” is used. Today a combination of historical observations and forecast data.
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="sunrise-channel">
+		<item-type>String</item-type>
+		<label>Sunrise</label>
+		<description>The formatted time of the sunrise (For example “2022-05-23T05:50:40”). See How to include sunrise,
+			sunset, moon phase, moonrise and moonset data into your API requests
+			(https://www.visualcrossing.com/resources/documentation/weather-api/how-to-include-sunrise-sunset-and-moon-phase-data-into-your-api-requests/)
+		</description>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="sunset-channel">
+		<item-type>String</item-type>
+		<label>Sunset</label>
+		<description>The formatted time of the sunset (For example “2022-05-23T20:22:29”). See How to include sunrise,
+			sunset,
+			moon phase, moonrise and moonset data into your API requests
+			(https://www.visualcrossing.com/resources/documentation/weather-api/how-to-include-sunrise-sunset-and-moon-phase-data-into-your-api-requests/)
```

Comment:
```suggestion
		<description>The formatted time of the sunset (For example “2022-05-23T20:22:29”).
```

Please remove these copy/paste text parts from the description as they are not from the binding perspective, but from the API perspective, that should be transparant to the user. (Now i noticed it, i also see this in the readme channel description table.